spec/spec.tex

```diff
@@ -1,4 +1,4 @@
-\documentclass[10pt,twoside,titlepage]{spec}
+\documentclass[10pt,oneside,titlepage]{spec}
```

Comment:
OK, I did that. It (still) looks good to me. I even printed out a random page...